<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Picaria</class>
 <widget class="QMainWindow" name="Picaria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Picaria</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/hole_empty.png</normaloff>Images/hole_empty.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background: url(':grid');
}</string>
   </property>
   <widget class="Hole" name="Hole00">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>0</number>
    </property>
    <property name="col" stdset="0">
     <number>0</number>
    </property>
    <property name="id" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole02">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>0</number>
    </property>
    <property name="col" stdset="0">
     <number>2</number>
    </property>
    <property name="id" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole04">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>0</number>
    </property>
    <property name="col" stdset="0">
     <number>4</number>
    </property>
    <property name="id" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole22">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>2</number>
    </property>
    <property name="col" stdset="0">
     <number>2</number>
    </property>
    <property name="id" stdset="0">
     <number>6</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole24">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>2</number>
    </property>
    <property name="col" stdset="0">
     <number>4</number>
    </property>
    <property name="id" stdset="0">
     <number>7</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole20">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>150</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>2</number>
    </property>
    <property name="col" stdset="0">
     <number>0</number>
    </property>
    <property name="id" stdset="0">
     <number>5</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole42">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>4</number>
    </property>
    <property name="col" stdset="0">
     <number>2</number>
    </property>
    <property name="id" stdset="0">
     <number>11</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole44">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>4</number>
    </property>
    <property name="col" stdset="0">
     <number>4</number>
    </property>
    <property name="id" stdset="0">
     <number>12</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole40">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>4</number>
    </property>
    <property name="col" stdset="0">
     <number>0</number>
    </property>
    <property name="id" stdset="0">
     <number>10</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>1</number>
    </property>
    <property name="col" stdset="0">
     <number>1</number>
    </property>
    <property name="id" stdset="0">
     <number>3</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole31">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>3</number>
    </property>
    <property name="col" stdset="0">
     <number>1</number>
    </property>
    <property name="id" stdset="0">
     <number>8</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>1</number>
    </property>
    <property name="col" stdset="0">
     <number>3</number>
    </property>
    <property name="id" stdset="0">
     <number>4</number>
    </property>
   </widget>
   <widget class="Hole" name="Hole33">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="icon">
     <iconset resource="Picaria.qrc">
      <normaloff>:/empty</normaloff>:/empty</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="row" stdset="0">
     <number>3</number>
    </property>
    <property name="col" stdset="0">
     <number>3</number>
    </property>
    <property name="id" stdset="0">
     <number>9</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJogo">
    <property name="title">
     <string>Picaria</string>
    </property>
    <addaction name="separator"/>
    <addaction name="n_game"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuModo">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="game09_buracos"/>
    <addaction name="game13_buracos"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutGame"/>
   </widget>
   <addaction name="menuJogo"/>
   <addaction name="menuModo"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="n_game">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Picaria.qrc">
     <normaloff>:/selectable</normaloff>:/selectable</iconset>
   </property>
   <property name="text">
    <string>New game</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="game09_buracos">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Picaria.qrc">
     <normaloff>:/selectable</normaloff>:/selectable</iconset>
   </property>
   <property name="text">
    <string>9 Holes</string>
   </property>
  </action>
  <action name="game13_buracos">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Picaria.qrc">
     <normaloff>:/selectable</normaloff>:/selectable</iconset>
   </property>
   <property name="text">
    <string>13 Holes</string>
   </property>
  </action>
  <action name="aboutGame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Picaria.qrc">
     <normaloff>:/selectable</normaloff>:/selectable</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="Picaria.qrc">
     <normaloff>:/selectable</normaloff>:/selectable</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Hole</class>
   <extends>QPushButton</extends>
   <header location="global">Hole.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Picaria.qrc"/>
 </resources>
 <connections/>
</ui>
